src/pushNotifications/TokenHandling.ts

```diff
@@ -38,7 +48,13 @@ const removeTokenFromServer = async (token: string) => {
     })
   };
 
-  if (accessToken) fetch(requestPath, fetchObj);
+  if (accessToken) 
+    // 204 means that it was a success on the server
+    // 404 means that the token was already not on the server and can be treated
+    //  as a success
+    return fetch(requestPath, fetchObj)
+      .then((response) => response.status === 204 || response.status === 404);
+  return false;
```

Comment:
please use {} for multiline body in ifs.
the `return false` at the end should also be separated by a line break.
this way now, the readability suffers.

```
if (accessToken) {
  // 204 means that it was a success on the server
  // 404 means that the token was already not on the server and can be treated
  //  as a success
  return fetch(requestPath, fetchObj)
    .then((response) => response.status === 204 || response.status === 404);
}
  
return false;
```